modules/container-registry/registry/main.bicep

```diff
@@ -185,6 +185,9 @@ param cMKKeyVersion string = ''
 @description('Conditional. User assigned identity to use when fetching the customer managed key. Note, CMK requires the \'acrSku\' to be \'Premium\'. Required if \'cMKKeyName\' is not empty.')
 param cMKUserAssignedIdentityResourceId string = ''
 
+@description('Optional. Array of Cache Rules.')
```

Comment:
@AlexanderSehr if we are enabling a preview feature, should this be advertised in the description? Just reminded me of the AKS cluster preview feature we recently merged.

Reply:
I guess we can. Just hope whoever owns this later takes note to remove it 😄

Reply:
Added a reference to the docs to the parameter metadata. The child itself already points it out in the description.